packages/remirror__extension-entity-reference/src/entity-reference-extension.ts

```diff
@@ -62,6 +62,7 @@ const createDecorationSet = (props: StateProps) => {
   defaultOptions: {
     getStyle: decorateEntityReferences,
     blockSeparator: undefined,
+    onClickMark: () => {},
```

Comment:
Could you pls add a unit test for this? I'm aware that this is testing likely the functionality of an underlying classes. Yet, I find it important because the users of the `EntityReferenceExtension` might rely on this functionality - and they don't care what provides it.

See here an example: https://github.com/remirror/remirror/blob/bb33bd813fe05f8d9feea21eb75fd7bb20ac8b63/packages/remirror__extension-link/__tests__/link-extension.spec.ts#L1042